### ISSUE TITLE
Improper packaging of the Framework NuGet package

### ISSUE THREAD

**Issue by rprouse** [nunit/nunit#977]
I have been working on getting the Xamarin runners ready for release, but they are not working with the current nuget package. I think the dotnet directory that I added to lib for DNX is conflicting with Android and iOS.

I need to double check my work in the Xamarin solution and make sure all the NuGet references are correct, but I wanted to enter this issue to inform the @nunit/core-team that there may be issues. AFAIK, If there is an issue, it is a nuget packaging issue, not a code issue.

I need to test creating the following from the NuGet package and make sure they all reference the correct assembly in the package.

- DNX test project
- Android test project,
- iOS test project
- WinPhone 8.1 test project

**Comment by rprouse** [comment]
It looks like this was a bug with NuGet upgrading the Beta 5 packages to the RC, probably because I changed the names of the portable directories in the package when DNX support was added and Silverlight support was dropped. This caused some of the upgraded references to be broken pointing at old directories. I uninstalled the NuGet packages and re-installed which fixed the references. I don't think end-users will run into this since the Xamarin runners are not released yet.

Android, iOS and Windows Phone 8.1 are now working. I will release those runners after a bit more testing. I am going to leave this issue open as a task for myself to test all supported platforms before the final 3.0 release to ensure everything works as expected.
